src/pages/auth/signup.js

```diff
@@ -58,7 +60,21 @@ export default function Login() {
 
         axios
             .post("http://localhost:3000/api/signup", payload)
```

Comment:
don't forget to put baseUrl in constant file, so other engineer can use it

Reply:
I think we can put the baseUrl to the .gitignore file for the next commit? After we change to localhost:3000 in local?